src/bootstrap/src/core/builder/mod.rs

```diff
@@ -363,18 +363,18 @@ impl PathSet {
         PathSet::Set(set)
     }
 
-    fn has(&self, needle: &Path, module: Kind) -> bool {
+    fn has(&self, needle: &Path, module: Kind, filter_start: bool) -> bool {
```

Comment:
I'm very wary of seeing cryptic booleans threaded through functions that are already have mysterious semantics to begin with.

This seems like it's going to cause a lot of frustration for whoever tries to properly clean up PathSet someday.